### PR TITLE
Feat/filter v2

### PR DESCRIPTION
**Pull request by asogwachiuba**
Filter fully functional and other small fixes

### CODE REVIEW

File: lib/core/services/event/event_service.dart [hunk 1]
```diff
@@ -116,20 +120,48 @@ class EventServiceImpl extends EventServices {
...
+    double? latitude,
+    double? longitue,
+    int? lowPriceRange,
+    int? highPriceRange,
+    String? categories,
+    String? startDate,
+    String? endDate,
+    bool? isPaid,
+  }) async {
     String urlMain = "https://ibloov-auth-staging.herokuapp.com/";
```
Comment (divincoder) [inline lib/core/services/event/event_service.dart:133]: Please remove this. not needed.

File: lib/features/tickets/widget/ticket_purchase_sucess/ticket_purchase_success_view.dart [hunk 2]
```diff
@@ -252,9 +254,9 @@ class TicketPurchasedSuccess extends ViewModelBuilderWidget<TicketPurchaseSucces
             ),
             SizedBox(height: 20),
             IbButton(
-              text: 'View Ticket(s)',
+              text: 'Continue',
               onPressed: () {
-                viewModel.gotoProfile();
+                viewModel.gotoProfile(userId: user.id);
```
Comment (divincoder) [inline lib/features/tickets/widget/ticket_purchase_sucess/ticket_purchase_success_view.dart:259]: Why are you passing a userId?

Comment (asogwachiuba) [reply lib/features/tickets/widget/ticket_purchase_sucess/ticket_purchase_success_view.dart:259]: If user id is null then its a guest user so it navigates to onboarding. If its not null its a signed in user so it navigates to home page.

Comment (divincoder) [reply lib/features/tickets/widget/ticket_purchase_sucess/ticket_purchase_success_view.dart:259]: I think it should navigateToExplore page incase they want to buy another ticket

File: lib/core/services/event/event_service.dart (hunk 1 above)
Comment (asogwachiuba) [reply lib/core/services/event/event_service.dart]: Thank you, i have corrected it.

File: lib/features/tickets/widget/ticket_purchase_sucess/ticket_purchase_success_view.dart (hunk 2 above)
Comment (asogwachiuba) [reply lib/features/tickets/widget/ticket_purchase_sucess/ticket_purchase_success_view.dart:259]: I followed what was said of the web version that it takes them to sign up after buying the tickets. But I could still change it now

**Comment by divincoder** [comment]
Resolve conflicts

**Comment by asogwachiuba** [comment]
> Resolve conflicts

I have resolved the conflict

File: lib/features/tickets/widget/ticket_purchase_sucess/ticket_purchase_success_view.dart (hunk 2 above)
Comment (divincoder) [reply lib/features/tickets/widget/ticket_purchase_sucess/ticket_purchase_success_view.dart:259]: ok

**Review by divincoder — APPROVED** [review]

**Pull request merged**